test/test-input/CMakeLists.txt

```diff
@@ -15,10 +25,4 @@ target_sources(
 
 target_link_libraries(test-input PRIVATE OBS::libobs)
 
-if(MSVC)
-  target_link_libraries(test-input PRIVATE OBS::w32-pthreads)
-endif()
```

Comment:
Is this transitively provided by OBS::libobs or something else now? Or is it no longer needed?